lib/puppet/defaults.rb

```diff
@@ -1450,6 +1450,11 @@ def self.initialize_default_settings!(settings)
 
         To turn off reports entirely, set this to `none`",
     },
+    :exclude_unchanged_resources => {
+      :default => false,
+      :type => :boolean,
+      :desc => 'TBD'
```

Comment:
I think the setting name makes sense from a user perspective "don't report on resources that haven't changed". But I think the JIRA ticket summary "Make unchanged events in report optional" is a little misleading, since we're excluding resource status objects and as opposed to the events contained within each resouce status.

Reply:
So...you think the name is good and I should update the JIRA ticket? Or should we finesse the name a bit more?

Reply:
Yeah I think the setting here is fine